File: production/client/src/main/scala/kpn/client/components/subset/routes/SubsetOrphanRouteFilterCriteria.scala
// TODO migrate to Angular
package kpn.client.components.subset.routes

import kpn.client.filter.TimeFilterKind

case class SubsetOrphanRouteFilterCriteria(
  broken: Option[Boolean] = None,
  lastUpdated: TimeFilterKind.Value = TimeFilterKind.ALL
)

File: client/src/app/analysis/pages/subset/orphan-routes/subset-orphan-route-filter-criteria.ts
import {TimestampFilterKind} from "../../../../kpn/filter/timestamp-filter-kind";

export class SubsetOrphanRouteFilterCriteria {
  constructor(readonly broken: boolean = null,
              readonly lastUpdated: TimestampFilterKind = TimestampFilterKind.ALL) {
  }
}
